test/show.jl

```diff
@@ -41,27 +39,181 @@ module TestShow
     refstr = """
     4×3 DataFrames.DataFrame
     
```

Comment:
Just a detail, but we probably don't need an empty line? That would be more consistent with the other format.

Reply:
removed